README.md

```diff
@@ -1,20 +1,53 @@
 # Android-Study-Jam
 All the material for the Android Study Jam at Women Techmakers Hamburg
 
-Everything here is work in progress and up to discussion!
+Everything here is **work in progress** and **up to discussion**!
 
 ## WTM-Messenger
 This is the reference app that we want to build during the course. The idea is a simple WhatsApp like chat application that uses Firebase as its backend.
 
 ### Proposed feature steps
 Not all must be accomplished
 
-1. Send end receive text messages and display them in a list.
+1. Send and receive text messages and display them in a list (Recycler View).
 
-2. Make users identifiable by a name
+2. Make users identifiable by a name (Firebase Authentication)
```

Comment:
I thought to first of all just let the user set a name somewhere and use it for the messages. The Firebase Authentication part was rather meant to be the login step.
I guess people should just play around with the Firebase DB first and later start to use different Firebase Services